.aws/deploy/support-task-definition.staging.json

```diff
@@ -0,0 +1,311 @@
+        {
+          "name": "NETLIFY_ACCESS_TOKEN",
+          "valueFrom": "STAGING_NETLIFY_ACCESS_TOKEN"
+        },
+        { "name": "NODE_ENV", "valueFrom": "STAGING_NODE_ENV" },
+        { "name": "OTP_EXPIRY", "valueFrom": "STAGING_OTP_EXPIRY" },
+        { "name": "OTP_SECRET", "valueFrom": "STAGING_OTP_SECRET" },
+        {
+          "name": "OUTGOING_QUEUE_URL",
+          "valueFrom": "STAGING_OUTGOING_QUEUE_URL"
+        },
+        { "name": "POSTMAN_API_KEY", "valueFrom": "STAGING_POSTMAN_API_KEY" },
+        {
+          "name": "POSTMAN_SMS_CRED_NAME",
+          "valueFrom": "STAGING_POSTMAN_SMS_CRED_NAME"
+        },
+        {
+          "name": "REDIRECT_URI",
+          "valueFrom": "STAGING_REDIRECT_URI"
+        },
+        {
+          "name": "SESSION_SECRET",
+          "valueFrom": "STAGING_SESSION_SECRET"
+        },
+        { "name": "SGID_CLIENT_ID", "valueFrom": "STAGING_SGID_CLIENT_ID" },
+        {
+          "name": "SGID_CLIENT_SECRET",
+          "valueFrom": "STAGING_SGID_CLIENT_SECRET"
+        },
+        {
+          "name": "SGID_REDIRECT_URI",
+          "valueFrom": "STAGING_SGID_REDIRECT_URI"
+        },
+        {
+          "name": "SGID_PRIVATE_KEY",
+          "valueFrom": "STAGING_SGID_PRIVATE_KEY"
+        },
+        {
+          "name": "SITE_CLONE_FORM_KEY",
+          "valueFrom": "STAGING_SITE_CLONE_FORM_KEY"
+        },
+        {
+          "name": "SITE_CREATE_FORM_KEY",
+          "valueFrom": "STAGING_SITE_CREATE_FORM_KEY"
+        },
+        {
+          "name": "SITE_LAUNCH_DYNAMO_DB_TABLE_NAME",
+          "valueFrom": "STAGING_SITE_LAUNCH_DYNAMO_DB_TABLE_NAME"
+        },
+        {
+          "name": "SITE_LAUNCH_FORM_KEY",
+          "valueFrom": "STAGING_SITE_LAUNCH_FORM_KEY"
+        },
+        {
+          "name": "SITE_PASSWORD_SECRET_KEY",
+          "valueFrom": "STAGING_SITE_PASSWORD_SECRET_KEY"
+        },
+        {
+          "name": "STAGING_SSH_PUBLIC_KEY",
+          "valueFrom": "STAGING_SSH_PUBLIC_KEY"
+        },
+        {
+          "name": "STAGING_SSH_PRIVATE_KEY",
+          "valueFrom": "STAGING_SSH_PRIVATE_KEY"
+        },
+        {
+          "name": "STEP_FUNCTIONS_ARN",
+          "valueFrom": "STAGING_STEP_FUNCTIONS_ARN"
+        },
+        {
+          "name": "SYSTEM_GITHUB_TOKEN",
+          "valueFrom": "STAGING_SYSTEM_GITHUB_TOKEN"
+        },
+        {
+          "name": "UPTIME_ROBOT_API_KEY",
+          "valueFrom": "STAGING_UPTIME_ROBOT_API_KEY"
+        },
+        {
+          "name": "SITE_CHECKER_FORM_KEY",
+          "valueFrom": "STAGING_SITE_CHECKER_FORM_KEY"
+        },
+        {
+          "name": "SITE_AUDIT_LOGS_FORM_KEY",
+          "valueFrom": "STAGING_SITE_AUDIT_LOGS_FORM_KEY"
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/aws/elasticbeanstalk/cms-backend-staging-node18/var/log/web.stdout.log",
```

Comment:
open to creating another log group, nbd

Reply:
think better to keep the same

Reply:
+1 to creating another log group

Maybe we can drop `elasticbeanstalk` from the name too...

Reply:
ya can, i think i'll change it to an ecs-prefixed log group **post tear down**  of eb